lib/bsdlib/bsdlib.c

```diff
@@ -28,8 +28,14 @@ static int _bsdlib_init(struct device *unused)
 	IRQ_DIRECT_CONNECT(BSD_NETWORK_IRQ, BSD_NETWORK_IRQ_PRIORITY,
 			   ipc_proxy_irq_handler, 0);
 
-	init_ret = bsd_init();
+	const bsd_init_params_t init_params =
+	{
+		.trace_on = true,                   //TODO: get the right definition from Kconfig
+		.bsd_memory_address = 0x20010000,   //TODO: get the right definition from partition manager
+		.bsd_memory_size = 0xc020           //TODO: get the right definition from partition manager
```

Comment:
I assume you will update before we merge this ?

Reply:
It depends when partition manager update is ready. Before that we need to pass these default values to bsd_init().

Reply:
got it. thanks for the info.